src/pages/news-announcements.tsx

```diff
@@ -91,9 +91,7 @@ const NewsAnnouncements = ({
               href="https://www.spaceforce.mil/News"
               target="_blank"
               rel="noreferrer noopener">
-              <button
-                type="button"
-                className="usa-button usa-button--accent-cool">
+              <button type="button" className="usa-button">
```

Comment:
The dark mode styles for this button were displaying on both light and dark mode, so thought I'd fix it.